README.md

```diff
@@ -1 +1,50 @@
-Random for initial commit
+# Watchtower Wallet and Operational Environment Simulator
+
+In a Revault deployment a watchtower (WT) is relied upon to enforce restrictive policies on funds which have been delegated to managers by stakeholders. To compute the operational costs and identify risks we have constructed a model of the WT wallet and its operating environment, and we simulate this for various types of Revault user.
```

Comment:
I think we should specify where these costs come from (the uncertainty of tx inclusion in a block, and to have funds at disposition in order to fee-bump)